Sources/SpeziLLMLocal/LLMLocalSession+Generate.swift

```diff
@@ -9,18 +9,18 @@
 import Foundation
 import MLX
 import MLXLLM
+import MLXLMCommon
 import MLXRandom
 import os
 import SpeziChat
 import SpeziLLM
 
 
 extension LLMLocalSession {
-    // swiftlint:disable:next identifier_name function_body_length
+    // swiftlint:disable:next identifier_name function_body_length cyclomatic_complexity
```

Comment:
Would be great if we can break it down into smaller private functions or extensions, e.g., handling the tokens. Could resolve most of the SwiftLint issues here and will make it nicer to maintain in the future.

Reply:
Separated logic for preparing model input and processing tokens into private functions.